src/config/webpack.config.js

```diff
@@ -103,11 +160,11 @@ module.exports = (env, argv) => {
         base(env, argv),
         {
           output: {
-            filename: `${pkg.name}.js`,
-            sourceMapFilename: `${pkg.name}.js.map`
+            filename: 'index.js',
+            sourceMapFilename: 'index.js.map'
```

Comment:
I think the package name is a good default, but we should make this configurable to resolve https://github.com/ipfs/aegir/pull/276 and allow JS IPFS to keep `index.js`.

Reply:
can we tackle this in another PR ? this keep the current behaviour and makes this PR easier to integrate. I need this for the bundle size PRs